src/component/term/TermDetail.tsx

```diff
@@ -205,9 +223,12 @@ export class TermDetail extends EditableComponent<
     const actions = [];
     if (!this.state.edit) {
       actions.push(
-        <IfNotInTerminalState term={this.props.term!}>
+        <IfNotInTerminalState
+          term={this.props.term!}
+          key="term.detail.edit.if.not.terminal"
+        >
           <IfVocabularyActionAuthorized
-            key="term-detail-edit"
+            key="term.detail.edit.if.authorized"
```

Comment:
Why change the key? And the change is not consistent, the remove action below has key `term.detail.remove.if` without the `.authorized` at the end.